arcade/experimental/video_cv2.py

```diff
@@ -110,27 +114,37 @@ def update(self, delta_time):
             exists, frame = self.video.read()
             if exists:
                 self.texture.write(frame)
+            # loop if we are at the end of the video
+            elif self.loop:
+                self.time = 0.0
+                self.video.set(cv2.CAP_PROP_POS_FRAMES, 0)
+
 
 
 class CV2PlayerView(arcade.View):
-    def __init__(self, path: str):
+    """A simple view to hold a video player using cv2.
+    requires opencv-python
```

Comment:
The formatting is a little off on this + you could be clearer about what opencv-python is.
```suggestion
    """
    A simple view to hold a video player using cv2.

    Requires the opencv-python module to be installed.
    """
```